readme.md

```diff
@@ -398,8 +401,6 @@ Running that with `node example.js` yields:
 
 remark lint rules *check* markdown.
 [`remark-stringify`][remark-stringify] (used in remark) *formats* markdown.
-When you configure lint rules and use remark to format markdown, you must
-manually synchronize their configuration:
```

Comment:
I think this statement is still correct. There is a new way to solve it though. That can be added as a new example. It is still important to call out the problem though.